frontend/src/pages/PredictedDataPage/PredictedDataPage.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Button, Container, List } from "@mui/material";
+import styles from "./PredictedDataPage.module.css";
+
+function PredictedFiles(): React.ReactElement {
```

Comment:
Function name should be the same as the file name.
```suggestion
function PredictedDataPage(): React.ReactElement {
```